canonical-websites/partners.ubuntu.com
# Remove OIL page, navigation and added redirect

## Done

1. remove the page
2. remove from the nav
3. remove OIL tested and Openstack from the databaes programme list
4. redirect /programmes/openstack to /programmes

## QA

1. Download this branch
2. `./run` the site
3. Go to http://0.0.0.0:8003/programmes/openstack
4. See that you are redirected to /programmes
5. see that /openstack isn't in the nav
6. on the live site, see that OIL tested and Openstack programmes are removed


## Issue / Card

Fixes #210

## Screenshot

![image](https://user-images.githubusercontent.com/441217/54084072-05668800-4324-11e9-8b6a-2e9957b78355.png)
